### ISSUE TITLE
Bulma dockset showing adverts in Dash (v6.0)

### ISSUE THREAD

**Issue by thedoglett** [Kapeli/Dash-User-Contributions#3169]
I've noticed that the Bulma docset is showing adverts in Dash (v6.0 - all paid for) and @Kapeli said to raise an issue here as its a user contributed docset (so here it is!).

This is what I'm seeing…

![Imgur](https://i.imgur.com/lVynm4x.jpg)

**Comment by godbout** [comment]
yes. i've let those on purpose. using Dash to view the docs is convenient but it shouldn't cut funding for those projects.
